web/cobrands/sass/_mixins.scss

```diff
@@ -176,8 +188,8 @@ $right: right;
   }
 
   background: -webkit-gradient(linear, 0% 0%, 0% $webkit-end-point, $webkit-gradient-stops);
-  @each $prefix in -moz-, -o-, -webkit-, "" {
-    background: #{$prefix}linear-gradient($stops);
+  @each $button-padding-rightefix in -moz-, -o-, -webkit-, "" {
```

Comment:
Another find-and-replace error here. I think maybe you just did a search for `$pr` and didn’t notice that it was also picking up the `$pr` at the start of `$property` and `$prefix`? Anyway, `$prefix` shouldn’t have been changed in these two lines.